source/actions3/actions/AMControlMoveActionInfo3.h

```diff
@@ -54,7 +54,7 @@ class AMControlMoveActionInfo3 : public AMActionInfo3
 	//////////////////////////
 
 	/// Returns a pointer to our move destination setpoint
-	const AMControlInfo* controlInfo() const { return &controlInfo_; }
+	AMControlInfo controlInfo() const { return controlInfo_; }
```

Comment:
Since this always a valid piece of memory, you could potentially make it a ```const AMControlInfo &```.  It's not a big deal though.